understanding/20/media-alternative-prerecorded.html

```diff
@@ -87,7 +87,7 @@ <h2>Benefits of Media Alternative (Prerecorded)</h2>
       
       <ul>
          
-         <li>People who cannot see well or at all and who also cannot hear well or at all can get
+         <li>People who are deaf-blind, who cannot see well or at all, and who also cannot hear well or at all can get
```

Comment:
"deaf-blind" is an existing English term that describes someone who is both deaf and blind, just like blind is a term that describes someone who cannot legally see. It's possible to rephrase as you state, but deaf-blind users have significantly different needs and need different solutions than users who are either deaf or blind users. In example, captions and audio descriptions are both useless to this group.

Reply:
I fully agree that deaf-blind users have other needs that deaf or blind people.
On the other hand, this Success Criterion talks about deaf-blind, blind, low vision, deaf,, and hard of hearing users, and I wondered whether this can be expressed shorter and sticking more to the wording inside other in-brief sections.

Reply:
Once the Understanding document includes considerations for those with cognitive disabilities, as captured by Rain and Gregg, it's going to add another group to the list.
I'm going to be stubborn on the existing wording for now. "More people" can include any of the users highlighted in the A and AA needs for captions and descriptions. But it is a game changer for deaf-blind users. It's worth highlighting their needs, IMO, while noting others also benefit.